contributing/diversity/governance.rst

```diff
@@ -0,0 +1,110 @@
+Any single veto will reject an actionable item.
+
+The purpose of having members with the right to veto is to prevent a “people’s
+majority” from overruling the core interests of the Symfony project. This will
+encourage communication between proposing members, the initiative’s key players
+and the Core Team to create realistic proposals, and in return any veto will
+come with a full explanation (not just a justification).
+
+Advice Process
+~~~~~~~~~~~~~~
+
+When a proposal on an actionable item is ready to be decided on, insight from
+the community (advice, general consensus, or non-binding poll) should be
+requested from the wider community - this will aim to include both those who
+will be meaningfully affected and those with meaningful expertise in the matter
+at hand.
+This feedback will enable the key players to have the confidence to vote for the
+best possible decision according to the information they have available, knowing
+that the responsibility they accept for said vote is justified.
+
+Voting
+~~~~~~
+
+Key players have the right to vote on proposals for actionable items.
+For an actionable item to pass, approval from greater than 50% of the voting key
+players is required. Use or management of finances/donations require at least a
+two-thirds majority to pass.
+The quorum required for votes to be considered valid is 75% of active, appointed
+key players.
+
+Guidance Principles
+-------------------
+
+Purpose
+~~~~~~~
+
+The initiative should be led by an effective team of key players that provide
+strategic guidance in line with the initiative’s aims and values, including a
+shared understanding with fellow key players to ensure that these are being
+delivered effectively and sustainably.
+
+Integrity
+~~~~~~~~~
+
+Key players should act with integrity: adopting values which help achieve the
+initiative’s purposes, even where difficult or unpopular decisions are required.
+Key players should undertake their duties, aware of the importance of confidence
+and trust in the initiative from the wider community, and ultimately
+acknowledges shared responsibility for the reputation of the Symfony project
+like the Core Team.
+
+Decision-making Effectiveness
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Key players should work as an effective team, using the appropriate balance of
+skills, experience, backgrounds and knowledge to make sure its decision-making
+processes are informed and equitable. Risk assessment and management systems
+should be set up and monitored.
+
+Openness and Accountability
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The behaviour and conduct of the initiative’s key players sets the tone for the
+rest of the community. Key players should lead by example to create a culture
+that enables members to feel it is safe to suggest, question and challenge -
+rather than avoid - difficult ideas and topics. Key players guide the initiative
+in being transparent, accountable and open.
```

Comment:
About transparency and accountability, could we mention explicitly above that all discussions must be done in publicly accessible mediums, such as public channels on Symfony's Slack?

Reply:
That's probably a good idea, but I'm not sure about enforcing it for certain things (an example would be Sage Sharp consulting which was discussed in today's meeting).
/cc @lsmith77

Reply:
also its simply not practical .. there can be DMs, there could even be calls or discussions at conferences. it could say that all discussions should be public as much as possible .. and of course we could mandate a public debate before a vote.